cmd/swapcli/main.go

```diff
@@ -945,6 +1037,50 @@ func runShutdown(ctx *cli.Context) error {
 	return nil
 }
 
+func runGetContractSwapInfo(ctx *cli.Context) error {
+	offerID, err := types.HexToHash(ctx.String(flagOfferID))
+	if err != nil {
+		return errInvalidFlagValue(flagOfferID, err)
+	}
+
+	c := newRRPClient(ctx)
+	resp, err := c.GetContractSwapInfo(offerID)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Contract address: %s\n", resp.SwapCreatorAddr)
+	fmt.Printf("Block at which newSwap was called: %d\n", resp.StartNumber)
+	fmt.Printf("Swap ID as stored in the contract: %s\n", resp.SwapID)
+	fmt.Printf("Swap struct as stored in the contract:\n")
+	fmt.Printf("\tOwner: %s\n", resp.Swap.Owner)
```

Comment:
Most terminals have tabs set to 8 space, which might be a little much here. In other `swapcli` output, like `query-all`, we are using 2-space indents.